edit-post/assets/stylesheets/main.scss

```diff
@@ -168,6 +168,32 @@ body.gutenberg-editor-page {
 			outline-offset: 0;
 		}
 	}
+
+	input[type=checkbox] {
+		border: 2px solid $dark-gray-300;
```

Comment:
That hardcoded border width makes me nervous. Do we want a rule that checkboxes have double the usual border width? I dunno.

Reply:
Hardcoded makes it sound so serious :) 

The double border is to give weight to the checkbox, without making it overwhelming. Visually I like the balance but design feedback may change this. What would be good CSS if/when the visuals are settled?

Reply:
Yeah that makes sense. I just think either a variable named 2px or even just using `$border-width + 1` would work? That's what this is... and it makes sense to base it off our global borders.